iphone/Maps/LocalizedStrings/el.lproj/Localizable.strings

```diff
@@ -217,8 +217,14 @@
 
 "pref_zoom_title" = "Πλήκτρα μεγέθυνσης";
 
-/* Settings «Map» category: «Night style» title */
-"pref_map_style_title" = "Νυχτερινή λειτουργία";
+/* Settings «Map» category: «Appearance» title */
+"pref_map_style_title" = "Εμφάνιση";
+
+/* Settings "Appearance" category: "Light" title */
```

Comment:
To be consistent between the two options, it should be

- Light: `Ανοιχτόχρωμη`
- Dark: `Σκουρόχρωμη`

or

- Light: `Ανοιχτή`
- Dark: `Σκούρα`